docs/wxm-devices/frequency-plans.mdx

```diff
@@ -0,0 +1,400 @@
+|--------------------|--------------|---------------|-----------------|------------------|
+| Kazakhstan         | KZ           | IN865         | EU868           | IN865            |
+| Kenya              | KE           | EU868         | EU868           | EU868            |
+| Kiribati           | KI           | -       | -         | -          |
+| Kosovo             | XK           | EU868         | EU868           | EU868            |
+| Kuwait             | KW           | EU868         | EU868           | EU868            |
+| Kyrgyzstan         | KG           | -         | -           | -            |
+
+### L
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Laos               | LA           | AS923_1       | US915           | AS923_1          |
+| Latvia             | LV           | EU868         | EU868           | EU868            |
+| Lebanon            | LB           | EU868         | EU868           | EU868            |
+| Lesotho            | LS           | EU433         | -            | Not supported           |
+| Liberia            | LR           | -       | -         | -          |
+| Libya              | LY           | -          | -           | -            |
+| Liechtenstein      | LI           | EU868         | EU868           | EU868            |
+| Lithuania          | LT           | EU868         | EU868           | EU868            |
+| Luxembourg         | LU           | EU868         | EU868           | EU868            |
+
+### M
+
+| Country            | Country Code | ISM Frequency |D1/M5/Pulse Freq. | H1/H2 Freq. |
+|--------------------|--------------|---------------|-----------------|------------------|
+| Macau             | MO           | AS923_1       | US915           | AS923_1          |
+| Madagascar         | MG           | EU868         | EU868           | EU868            |
+| Malawi             | MW           | EU868         | EU868           | EU868            |
+| Malaysia           | MY           | AS923_1       | US915           | AS923_1          |
+| Maldives           | MV           | EU868         | EU868           | EU868            |
+| Mali               | ML           | EU433       | -           | Not supported            |
+| Malta              | MT           | EU868         | EU868           | EU868            |
+| Marshall Islands   | MH           | -     | -           | -            |
+| Martinique         | MQ           | EU868         | EU868           | EU868            |
+| Mauritania         | MR           | EU868         | EU868           | EU868            |
+| Mauritius          | MU           | EU433         | -           | Not supported            |
+| Mayotte            | YT           | EU868         | EU868           | EU868            |
+| Mexico             | MX           | AU915_SB2     | US915           | AU915            |
```

Comment:
Mexico is wrong, it has the following attributes:
Mexico | US915 | US915 | US915 |